modules/gtk/hm.nix

```
@@ -29,25 +29,66 @@ in {
         window.background { border-radius: 0; }
       '';
     };
+
+    flatpakSupport.enable = lib.mkEnableOption "support for theming Flatpak apps";
```

> ```patch
> From 5331b3ccea85b94f729c40db416db8c4354ece6c Mon Sep 17 00:00:00 2001
> From: bricked <git@bricked.dev>
> Date: Sun, 22 Dec 2024 23:29:35 +0100
> Subject: [PATCH 1/3] gtk: add support for theming flatpak apps
>
> This support can be enabled using `targets.gtk.flatpakSupport.enable`.
> It is likey incompatible with other modules that write to this file and
> is therefore disabled by default.
> ```

For consistency, we should use `config.lib.stylix.mkEnableTarget`:

```suggestion
    flatpakSupport.enable =
      config.lib.stylix.mkEnableTarget "support for theming Flatpak apps" false;
```

To improve discoverability and make Stylix mostly "just work", it might be better to enable this feature by default:

```suggestion
    flatpakSupport.enable =
      config.lib.stylix.mkEnableTarget "support for theming Flatpak apps" true;
```

Let's optimistically assume that people affected with nix-flatpak collisions, are able to find our `flatpakSupport.enable` option. In that case, everyone else would just benefit from the Flatpak support.